pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -718,11 +707,11 @@ def format_nested_dict(d: dict) -> dict:
         return format_nested_dict(model_config.copy())
 
 
-class DelayedSaturatedMMM(
+class Philly(
```

Comment:
I suggest we keep the class DelayedSaturatedMMM and add a deprecation warning. We can say we will remove it in the next release (e.g. 0.6)